.devcontainer/devcontainer.json

```diff
@@ -8,7 +8,7 @@
 
 	// These extensions are loaded for all users by default.
 	"extensions": [
-		"matklad.rust-analyzer",
+		"rust-lang.rust-analyzer",
```

Comment:
the rust plugin doesn't work without this change. I can make this change separately if desired.